src/components/seachContainer/SearchContainer.tsx

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import './searchContainer.scss';
+import { useSelector } from 'react-redux';
+import { RootStore } from '../../store';
+
+const SearchContainer = () => {
+  const jobState = useSelector((state: RootStore) => state.job);
+
+  return (
+    <div
+      id='searchContainer'
+      className=' container-fluid px-5 mb-2 bg-dark text-white'
+    >
+      <h3 className='px-5 pt-5 pb-3 text-white fs-2 text-wrap'>
+        {jobState.job?.length} IT Jobs For Chất Developers
+      </h3>
+      <div className='p-5 text-white fs-3 row g-3 form'>
+        <div className=' col-md-8'>
+          <input
+            className='form-control form-input'
+            type='text'
+            placeholder='Keyword skill (Java, iOS...), Job Title, Company...'
+          />
+        </div>
+        <div className='col-lg-2'>
+          <select
+            defaultValue={'default'}
+            name=''
+            id=''
+            className='form-select'
+          >
+            <option value='default' disabled>
+              All Cities
+            </option>
+            <option value='1'>Ho Chi Minh</option>
+            <option value='2'>Ha Noi</option>
+            <option value='3'>Da Nang</option>
```

Comment:
Please create static options and use map functionality for this.

Reply:
I missed this issue, will add it in the next commit